modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less

```diff
@@ -1,7 +1,12 @@
 .module {
     h1 {
         margin-top: 0;
-        margin-bottom: 20px;
+        color: @navbarInverseBackgroundHighlight;
+        font-family: "Source Sans Pro","Helvetica Neue",Arial,sans-serif34;
```

Comment:
Maybe also in here we could use some variable for font-definition (actually there might be some in `variables/fonts.less`)